gui-backend/src/test/java/com/rln/gui/backend/implementation/TransactionsApiImplTest.java

```diff
@@ -179,7 +179,7 @@ void apiTransactionsGetRejected() throws InvalidProtocolBufferException {
     cleanup(getCurrentBankPartyId(), RejectedTransferProposal.TEMPLATE_ID, rejectedCid);
   }
 
-  private void cleanup(Party partyId, Identifier identifier, String contractId)
+  public static void cleanup(Party partyId, Identifier identifier, String contractId)
```

Comment:
Why was this necessary?

Reply:
I wanted to use the same method in BalancesApiImplTest.

Reply:
Then can we name it properly and move it to `LedgerBaseTest` or some utility class? E.g. `cleanupContract` or `archive`.

Reply:
okay, LedgerBaseTest